pkg/security/securitycontextconstraints/selinux/mustrunas_test.go

```diff
@@ -80,42 +86,158 @@ func TestMustRunAsValidate(t *testing.T) {
 	seType.Type = "invalid"
 
 	tests := map[string]struct {
-		seLinux     *api.SELinuxOptions
-		expectedMsg string
+		seLinux         *api.SELinuxOptions
+		expectedSeLinux *api.SELinuxOptions
+		expectedMsg     string
 	}{
 		"invalid role": {
-			seLinux:     role,
-			expectedMsg: "does not match required role",
+			seLinux:         role,
+			expectedSeLinux: newValidOpts(),
+			expectedMsg:     "does not match required role",
 		},
 		"invalid user": {
-			seLinux:     user,
-			expectedMsg: "does not match required user",
+			seLinux:         user,
+			expectedSeLinux: newValidOpts(),
+			expectedMsg:     "does not match required user",
 		},
 		"invalid level": {
-			seLinux:     level,
-			expectedMsg: "does not match required level",
+			seLinux:         level,
+			expectedSeLinux: newValidOpts(),
+			expectedMsg:     "does not match required level",
 		},
 		"invalid type": {
-			seLinux:     seType,
-			expectedMsg: "does not match required type",
+			seLinux:         seType,
+			expectedSeLinux: newValidOpts(),
+			expectedMsg:     "does not match required type",
+		},
+		"invalid level, expected sensitivity with a wrong prefix": {
+			seLinux:         newValidOpts(),
+			expectedSeLinux: newValidOptsWithLevel("s0-w2"),
+			expectedMsg:     "does not match required level",
+		},
+		"invalid level, actual sensitivity with a wrong prefix": {
+			seLinux:         newValidOptsWithLevel("s0-w2"),
+			expectedSeLinux: newValidOpts(),
+			expectedMsg:     "does not match required level",
+		},
+		"invalid level, sensitivity only vs sensitivity with a category": {
+			seLinux:         newValidOptsWithLevel("s1"),
+			expectedSeLinux: newValidOptsWithLevel("s1:c1"),
+			expectedMsg:     "does not match required level",
+		},
+		"invalid level, expected category with a wrong prefix": {
+			seLinux:         newValidOpts(),
+			expectedSeLinux: newValidOptsWithLevel("s0:c1.w2"),
+			expectedMsg:     "does not match required level",
+		},
+		"invalid level, actual category with a wrong prefix": {
+			seLinux:         newValidOptsWithLevel("s0:c1.w2"),
```

Comment:
Isn't this the same as https://github.com/openshift/origin/pull/16432/files#diff-7116cbe02d68c6308574bed3cf7fb864R130

Reply:
The values are the same but in the first case they are in expected category and in another case in an actual category. I was looking on the code coverage and these 2 tests cover all the possible branches. Let me know if you think that 2 tests are superfluous, I can remove one of them, if you want.

Reply:
oh oh, I wrong. I see now. Sorry.